lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -367,6 +367,7 @@ public static function fileUploadFunction(array $params, $template)
         $multiple = isset($params['multiple']) ? ConfigHelper::checkValue($params['multiple']) : true;
 
         $image_resize = !isset($params['image_resize']) || !empty($params['image_resize']);
+        $view_containerid = isset($params['view_containerid']) ? $params['view_containerid'] : '';
```

Comment:
Tu nie warto ograniczać się do komiecznoóści wskazania ID elementu prezentującego obiekt URL - lepiej zrobić z tego view_selector i w formacie css selectora móc wskazywać element, gdzie będzie prezentowany podgląd.

Reply:
Nie ma konieczności wskazywania. 

Założenie moje było takie:
- gdy podamy ID selektora w to podglądany plik wyświetla się w nim,
- gdy nie podamy ID selectora to plik pokazuje się w dialogowym popupie,